rest_test/hooks.go

```diff
@@ -0,0 +1,378 @@
+package main
```

Comment:
If we're not using hooks I think we should remove this file. It will always be in the git history if we want it back in the future.

Reply:
🆗